vignettes/A1_Installation.Rmd

```diff
@@ -107,6 +107,221 @@ are:
 
 ## Mac (**NOT** required for this course)
 
+### Install Xcode and Xcode command line tools
+Make sure you have Xcode and Xcode command line tools installed on your mac. If not,
+you can run the following command on your terminal application to install Code
+
+```{bash xcode-install, eval=FALSE}
+xcode-select --install
+```
+
+For command line tools, go to http://developer.apple.com/downloads and sign in with
+your Apple ID (the same one you use for iTunes and app purchases). Look for 'Command line tools'
+through the search box, download the required file and install via the `.dmg` file on your machine.
+
+For more detailed instructions on this, please visit [this link](https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/#laptop-script).
+
+### Install of R and RStudio using homebrew (easy!)
+
+Open your terminal application on your mac. Follow the steps below:
+
+* Install [homebrew](http://brew.sh/) on your machine, copy and paste this in your terminal.
+
+```{bash hombrew-install, eval=FALSE}
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+* Update the "taps" on your machine and check if everything is fine.  
+
+```{bash brew-update, eval=FALSE}
+brew update && brew doctor
+```
+
+* Get brew cask and the homebrew science taps
+
+```{bash brew-cask, eval=FALSE}
+# Brew taps and requirements
+brew tap caskroom/cask
+brew tap homebrew/science
+brew install caskroom/cask/brew-cask
+```
+
+* Install Xquartz and Java
+
+```{bash xquartz-java, eval=FALSE}
+# Dependencies
+brew cask install xquartz
+brew cask install java
+```
+
+* Finally install R and Rstudio
+
+```{bash install-r, eval=FALSE}
+# Install R with X11
+brew install --with-x11 r
+
+# Install RStudio
+brew cask install --appdir=/Applications rstudio
+```
+
+Note: the inclusion of the `--appdir` option. By default, Cask formula’s will
```

Comment:
The apostrophe in "Cask's formula" is a 'fancy' (non-ASCII) quote, which requires that the vignettee declare an encoding. I replaced it by a straight quote.